compose-mac.yaml

```diff
@@ -121,10 +138,24 @@ services:
       - ASPNETCORE_ENVIRONMENT=Development
       - FUNCTIONS_WORKER_RUNTIME=dotnet-isolated
       - ASPNETCORE_URLS=http://*:6009
-      - GetEpisodeUrl=http://get-episode-mgmt:6060/api/GetEpisode
       - GetParticipantUrl=http://get-participant:6061/api/GetParticipant
-      - CreateParticipantScreeningEpisodeUrl=http://create-participant-screening-episode:6010/api/CreateParticipantScreeningEpisode
       - CreateParticipantScreeningProfileUrl=http://create-participant-screening-profile:6011/api/CreateParticipantScreeningProfile
```

Comment:
Same here

Reply:
Same as above, calling the Data service one